src/pages/docs/creators/ray/quickstart.md

```diff
@@ -0,0 +1,100 @@
+---
+description: quick and easy, copy and run instructions on launching and decommissioning the Ray on Golem cluster
+title: Ray on Golem QuickStart
+type: guide 
+---
+
+# Quickstart
+
+This quickstart shows you how to set Ray and Ray on Golem up, start your cluster, test it, and then stop it.
+It limits the explanation to the bare minimum - if you are looking for more details jump to [setup tutorial](/docs/creators/ray/setup-tutorial)
+
+
+## Install software
+
+The first step is installing Ray and Ray on Golem (recommended within a clean virtual environment)
+
+```bash
+# install ray & ray-on-golem
+pip install -U ray[default] ray-on-golem
```

Comment:
installing `ray-on-golem` already installs ray for you

Reply:
yup, but we still want to show to the user that they are using ray - we don't want to hide it underneath golem stuff